docs/elasticsearch-specifications.md

```diff
@@ -0,0 +1,31 @@
+# Elasticsearch Specifications
+
+## Supported Versions
+
+The following versions of Elasticsearch are supported:
+
+- 6.2
+- 6.0
+- 5.5
+- 5.3
+- 5.1
+- 2.3
+- 1.5
+
+It's recommended to use the most up-to-date version. The specific version you want to use should be communicated to the Human Made team when requesting Elasticsearch be enabled.
```

Comment:
Altis team? Platform team? Support team?

Reply:
Yeah not sure, I think we'll correct once we go ahead with the name